### PR TITLE
Somewhat serious

### PR DESCRIPTION
**Pull request by gogogarrett**
I think it would be funny to have this, thought? flame?

### CODE REVIEW

**Comment by parndt** [comment]
I'm not sure.. what would this do to code quality to have spurious constants?

**Comment by parndt** [comment]
Also squash your commits :P also don't use a fork just open a branch on the main repo!

**Comment by knewter** [comment]
I'm for it but as a commented out constant.

File: lib/abc.rb [hunk 1]
```diff
@@ -2,4 +2,5 @@
 # Abc is the Afterburner CMS namespace.
 
 module Abc
+  ITS_AS_EASY_AS = 123
```
Comment (robyurkowski) [inline lib/abc.rb:5]: Shouldn't this actually be a variable? What if the current release is _not_ as easy as 123? Maybe it's approaching 60 or so?

**Comment by robyurkowski** [comment]
I keep wavering back and forth on this one, and I think what finally settles it is that this is kind of unprofessional. Not in a bad way -- and in a way that makes me laugh... but if we do want this to be a serious CMS, I do think we need to be very careful to avoid humour for humour's sake.

I'm okay being funny in the course of doing our business, but I don't think this actually adds anything that we (or anyone else) will ever use, and it does increase the complexity, however negligible. Let's be funny where it's appropriate... but let's keep our commits to moving us forward.

**Comment by GeekOnCoffee** [comment]
I'd counter that argument with: 
https://github.com/rails/rails/blob/master/activesupport/lib/active_support/core_ext/array/access.rb#L50-L53

**Pull request closed without merging**